redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -463,6 +463,32 @@ object Input {
     def encode(data: Update): Chunk[RespValue.BulkString] = Chunk.single(encodeString(data.stringify))
   }
 
+  case object GetExPersistInput extends Input[(String, Boolean)] {
+    override private[redis] def encode(data: (String, Boolean)): Chunk[RespValue.BulkString] =
+      if (data._2) Chunk(encodeString(data._1), encodeString("PERSIST")) else Chunk(encodeString(data._1))
+  }
+  case object GetExInput extends Input[(String, Expire, Duration)] {
+    override private[redis] def encode(data: (String, Expire, Duration)): Chunk[RespValue.BulkString] =
+      data._2 match {
+        case Expire.SetExpireSeconds =>
+          Chunk(encodeString(data._1), encodeString("EX")) ++ DurationSecondsInput.encode(data._3)
+        case Expire.SetExpireMilliseconds =>
+          Chunk(encodeString(data._1), encodeString("PX")) ++ DurationMillisecondsInput.encode(data._3)
+        case _ => Chunk(encodeString(data._1))
+      }
+  }
+
+  case object GetExAtInput extends Input[(String, ExpiredAt, Instant)] {
+    override private[redis] def encode(data: (String, ExpiredAt, Instant)): Chunk[RespValue.BulkString] =
+      data._2 match {
+        case ExpiredAt.SetExpireAtSeconds =>
+          Chunk(encodeString(data._1), encodeString("EXAT")) ++ TimeSecondsInput.encode(data._3)
+        case ExpiredAt.SetExpireAtMilliseconds =>
+          Chunk(encodeString(data._1), encodeString("PXAT")) ++ TimeMillisecondsInput.encode(data._3)
+        case _ => Chunk(encodeString(data._1))
+      }
```

Comment:
Can we pattern match on the whole `data` tuple? This way we would not need this hard to read `_2`, `_1` etc but it's just cosmetics.

Reply:
Thanks